### ISSUE TITLE
320 images breaking gui

### ISSUE THREAD

**Issue by rjp5nc** [awedwards/daphnia#16]
There seem to be ~320 images that the gui does not like. This was what had caused the crash the other day, not the lack of a stage micrometer. It appears that they seem to be missing quite a bit of information. Could you double check why they are nan's, and why the gui pipeline can't load them in? If thats a huge issue, we can definitely exclude them, but we would like you to check first, because that is over 300 images that we would have to exclude. 

![screen shot 2018-10-17 at 10 15 23 am](https://user-images.githubusercontent.com/43250118/47092844-e1c86700-d1f5-11e8-8e49-22e20c468186.png)

**Comment by rjp5nc** [comment]
This is the actual error it gives, when trying to load it in:

![screen shot 2018-10-17 at 1 07 16 pm](https://user-images.githubusercontent.com/43250118/47103607-991ca800-d20d-11e8-8d78-0c2b4e18d2a0.png)

**Comment by awedwards** [comment]
Ah, ok that helps. My guess is that the automatic pipeline was not able to fit points to these images.

So the way the GUI is set up now, you can only fix images, rather than fit a new line to an image (which you would have to do for these 300). I can rework the GUI to handle these, but it will take some work.

**Comment by beckerdoerthe** [comment]
Thanks for clarification, Austin. I think, we don't need to make this our primary focus now. I will simply exclude those 300+ pics from our current input files. But if you - at some point - can think of an easy fix for this that would be great!

**Comment by beckerdoerthe** [comment]
One more question re this though. Austin, what is the criterion for passing the automated PF?  I would have assumed that only pictures that we were able to fit a shape to would get a "P". However, those 300+ pics that Robert mentions above all have a "P" in this column.

**Comment by awedwards** [comment]
Did this get resolved?

**Comment by rjp5nc** [comment]
Yeah, I think the original issue was when we thought it was crashing due to the lack of a stage micrometer, which wasn't actually the issue. Certain pictures seemed to crash the gui, but they don't seem to anymore. I ran through one of the sets of 50 which had a number of images that used to crash it, and was able to make it all the way through. It had only skipped two images that we did not actually have the images for.
